OpenXmlPowerTools/ListItemTextGetter_de_DE.cs

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OpenXmlPowerTools
+{
+    public class ListItemTextGetter_de_DE
+    {
+        private static string[] OneThroughNineteen = {
+            "eins", "zwei", "drei", "vier", "fünf", "sechs", "sieben", "acht",
+            "nuen", "zehn", "elf", "zwölf", "dreizehn", "vierzehn",
```

Comment:
please replace `nuen` with `neun` everywhere.

Reply:
Ok, I fixed that.